### PR TITLE
Create a proposal template.

### PR DESCRIPTION
**Pull request by bobbyrullo**
Also update CONTRIBUTING.md to mention use of proposals

### CODE REVIEW

**Comment by bobbyrullo** [comment]
@sym3tri PTAL

**Comment by bobbyrullo** [comment]
@sym3tri ping

File: PROPOSAL_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+Describe the logical data model that your proposal adds.
+
+## Data Storage
+
+Describe how the data will be persisted.
+
+## API
+
+Describe the methods that the API will expose.
```
Comment (sym3tri) [inline PROPOSAL_TEMPLATE.md]: and any breaking changes

**Comment by sym3tri** [comment]
1 comment, otherwise LGTM

**Comment by chris-rock** [comment]
+1

**Pull request merged**